src/test/java/org/tugraz/sysds/test/functions/builtin/BuiltinMiceNominalTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 Graz University of Technology
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.tugraz.sysds.test.functions.builtin;
+
+import org.junit.Test;
+import org.tugraz.sysds.common.Types;
+import org.tugraz.sysds.lops.LopProperties;
+import org.tugraz.sysds.test.AutomatedTestBase;
+import org.tugraz.sysds.test.TestConfiguration;
+
+public class BuiltinMiceNominalTest extends AutomatedTestBase {
+	private final static String TEST_NAME = "mice_nominal";
+	private final static String TEST_DIR = "functions/builtin/";
+	private static final String TEST_CLASS_DIR = TEST_DIR + BuiltinMiceNominalTest.class.getSimpleName() + "/";
+
+	private final static String DATASET = SCRIPT_DIR+"functions/transform/input/homes3/homes.csv";
+	private final static int iter = 3;
+	private final static int com = 2;
+
+	@Override
+	public void setUp() {
+		addTestConfiguration(TEST_NAME, new TestConfiguration(TEST_CLASS_DIR, TEST_NAME, new String[]{"B"}));
+	}
+
+	@Test
+	public void testMiceNominalCP() {
+		runMiceNominalTest(LopProperties.ExecType.CP);
+	}
+
+	@Test
+	public void testMiceNominalSpark() {
+		runMiceNominalTest(LopProperties.ExecType.SPARK);
+	}
+
+	private void runMiceNominalTest(LopProperties.ExecType instType) {
+		Types.ExecMode platformOld = setExecMode(instType);
+		try {
+			loadTestConfiguration(getTestConfiguration(TEST_NAME));
+			String HOME = SCRIPT_DIR + TEST_DIR;
+			fullDMLScriptName = HOME + TEST_NAME + ".dml";
+			programArgs = new String[]{"-nvargs", "X=" + DATASET, "iteration=" + iter, "com=" + com, "data=" + output("B")};
+
+			runTest(true, false, null, -1);
```

Comment:
we should extend theses tests to check the results as well (maybe in a certain range).